README.md

```diff
@@ -13,6 +13,20 @@ library" instead.
 
 _Instructions for getting up and running go here._
 
+## Examples
+
+Run the [examples](examples) in this repo with:
+
+```sh
+yarn serve-examples
+```
```

Comment:
I think we should call out that `yarn build-examples` needs to be run first. I didn't do this and ran into problems, and was confused until I looked at the diff for `package.json`

Reply:
:thinking: It _shouldn't_ need to be run first. I'll try some things...

Reply:
Just confirmed: I deleted `examples/webpack-output` for good measure and ran `yarn serve-examples` and it worked as expected. Which makes sense since the webpack dev server middleware doesn't actually emit the output to `webpack-output`, nor does it use it. :man_shrugging:

Reply:
I wrote this not knowing that I was missing your local changes :upside_down_face: and noticed that I was getting errors about files not existing which `build-examples` resolved. Things are working now!